src/wgpu_engine.rs

```diff
@@ -644,7 +644,10 @@ impl WgpuEngine {
             layout: Some(&compute_pipeline_layout),
             module: &shader_module,
             entry_point: "main",
-            compilation_options: PipelineCompilationOptions::default(),
+            compilation_options: PipelineCompilationOptions {
+                zero_initialize_workgroup_memory: false,
+                ..PipelineCompilationOptions::default()
```

Comment:
Do we have a policy of whether using `..Default::default()` or `..TheWholeLongStructName::default()` ?

Reply:
I personally go for the former.

Also, as a point of curiosity, less noisy default syntax was one of the things discussed at the unconference for how the Rust language could better support UI and application programming.

Reply:
Only thing I know is that there was a [rust internals discussion](https://internals.rust-lang.org/t/add-default-keyword/20810/13) about a `default` keyword like in C#

Reply:
oh wait I can also put it here :0 github actually surprises me sometimes
```suggestion
                ..Default::default()
```